.circleci/config.yml

```diff
@@ -227,7 +227,7 @@ jobs:
     - store_artifacts:
         path: ~/dist/cli
 
-  build-desktop-linux:
+  buil-tray-linux:
```

Comment:
Small typo here but not a show stopper. Though, do we want to add these jobs to the workflow?

Reply:
Was going with the etherem crowd... naw i'll fix that, and yes i'll add it to workflow 👍